src/main/play-doc/developer/CreatingBundles.md

```diff
@@ -84,23 +84,23 @@ Typesafe config provides the ability to substitute environment variables if they
 
 
 ```scala
-BundleKeys.endpoints := Map("customer-service" -> Endpoint("http", services = Set(URI("http://:5444/customers"))))
+BundleKeys.endpoints := Map("customer_service" -> Endpoint("http", 0, "customers", RequestAcl(Http("^/customers".r))))
```

Comment:
Well, we use the `-` in endpoints and configuration files since the beginning. I feel that this (loose) coding convention which most users use in names and configuration files. So I wouldn't change the `-` to `_`.

The coding convention for system environment variables is to use `_` instead.

If we want we can add a sentence that we create environment variables with `_`, even if the endpoint name uses a `-`. This sentence should then part of http://conductr.lightbend.com/docs/1.2.x/BundleEnvironmentVariables.